src/project.js

```diff
@@ -32,7 +32,13 @@ const conditions = [
   },
 ];
 
-function project(location) {
+const height = types.heightLevel
+  .map(h => h.key)
+  .splice(0, 7)
```

Comment:
Why do you splice?

Reply:
beacuse splicing make sures that the first heightlevel is Obersubalpin. Otherwise the list would start from unter-/obermontan and recursion would happen 4 more times which is unnecessary.

Reply:
Ok, understand. To me it looks like our heightlevel metadata is not optimal. Let's keep it like this, because I guess we need to refactor this a bit based on the new spatial data anyway.